app/services/fraud_review_check.rb

```diff
@@ -0,0 +1,29 @@
+class FraudReviewCheck
+  class NullFraudReviewUser
+    def fraud_review_pending_profile; end
+
+    def fraud_rejection_profile; end
+  end
+
+  attr_reader :user
+
+  def initialize(user)
+    @user = user || NullFraudReviewUser.new
+  end
+
+  def fraud_check_failed?
+    fraud_review_pending? || fraud_rejection?
+  end
+
+  def fraud_review_pending?
+    user.fraud_review_pending_profile.present?
+  end
+
+  def fraud_rejection?
+    user.fraud_rejection_profile.present?
+  end
+
+  def fraud_review_eligible?
+    user.fraud_review_pending_profile&.fraud_review_pending_at&.after?(30.days.ago)
+  end
```

Comment:
Since this logic is here, can the [similar methods](https://github.com/18F/identity-idp/blob/c8d8a3829aba4c7fd444e7d9e4f5a55b1844e12b/app/models/user.rb#L117-L127) be removed from User? Or is that a followup PR?

Reply:
That is a follow-up PR for when I get the decision tooling wired up